### PR TITLE
 Fix: sticky header, sticky table

### PR DESCRIPTION
**Pull request by nthgness**
# Template PR swap.react

If the build is not going, the PR do not even look

* Match style guide 😏
  - [ ] [js](https://github.com/airbnb/javascript#ecmascript-6-es-2015-styles)
  - [ ] [react](https://github.com/airbnb/javascript/tree/master/react)
  - [ ] [css/sass](https://github.com/airbnb/css)
* In the description issue 😋
  - [ ] number issue
  - [ ] brief description of the work
  - [ ] why they chose this solution
* Git 😈
  - [ ] [The rebase master is done](https://git-scm.com/book/en/v2/Git-Branching-Rebasing)
  `git pull --rebase origin master`

For the correct PR bonus tokens ✨ <br>
💔Thank you!

### CODE REVIEW

File: shared/components/tables/Table/Table.js [hunk 1]
```diff
@@ -25,24 +28,26 @@ export default class Table extends React.Component {
   }
 
   handleScrollTable = () => {
-    const { id } = this.props
     let scrollTop = window.pageYOffset || document.documentElement.scrollTop
+    let tableOffset = document.getElementById(this.props.id).offsetTop
+    let tableHeight = document.getElementById(this.props.id).clientHeight
```
Comment (nikdementev) [inline shared/components/tables/Table/Table.js]: в React так не делается надо через refs

**Comment by nthgness** [comment]
Изменил на refs

File: shared/components/tables/Table/Table.js [hunk 2]
```diff
@@ -29,8 +30,8 @@ export default class Table extends React.Component {
 
   handleScrollTable = () => {
     let scrollTop = window.pageYOffset || document.documentElement.scrollTop
-    let tableOffset = document.getElementById(this.props.id).offsetTop
-    let tableHeight = document.getElementById(this.props.id).clientHeight
+    let tableOffset = ReactDOM.findDOMNode(this.table).offsetTop
+    let tableHeight = ReactDOM.findDOMNode(this.table).clientHeight
```
Comment (nikdementev) [inline shared/components/tables/Table/Table.js]: внутри класса к ним можно просто через this.table.element обращаться
использование ReactDOM.find тоже не рекомендуется

**Comment by nikdementev** [comment]
LGTM !

**Comment by nthgness** [comment]
: )

**Pull request merged**